### PR TITLE
Add unit test for empty arrays.

### PR DESCRIPTION
**Pull request by tongfa**
| Type       | Description                                                                | Icon | Changelog |
| ---------- | ---------------------------------------------------------------------------| :--: | :-------: |
| `test`      | Making sure we don't break things later                                   |  📌  |           |
-------------------------------------------------------------------------------------------------------------

### 🛠️ Changes:
* added a unit test to verify that empty arrays are propagated through the reactivity system when the first item is added.

### 📢 Additional Info:
This PR did not meet my hopes and dreams.  I was hoping to write a unit test that fails when 3899285aed7a81c4a5845686d63a71b58340ca83 is reverted.  But this test still passes.  None the less I think it's a valid unit test.

### 📚 Bookkeeping:

**Testing (if applicable)**:
- [X] Ran/wrote unit tests for this

**Checklist**
- [X] Assigned PR to myself
- [X] Added at least 1 person on the team as reviewer
- [NA] Release Notes: PRs types that have the :spiral_notepad: next to them also require release notes to be added to the `CHANGELOG.md`

### CODE REVIEW

File: src/integrations/__spec__/testAccess.ts [hunk 1]
```diff
@@ -170,6 +170,28 @@ export default function testClass(integrationName, integration) {
         expect(typeof Madrone.lastAccessed(person.personInner)).toEqual('number');
         expect(typeof Madrone.lastAccessed(personInner)).toEqual('number');
       });
+
+      test('computed that depends on reactive array that starts out with zero length', async () => {
```
Comment (treardon17) [inline src/integrations/__spec__/testAccess.ts:174]: Looks like the other tests in this file are specifically testing the `lastAccessed` method. I think this test would be a better candidate for `testClass.ts`

**Review by treardon17 — COMMENTED** [review]
Thanks for the extra test! Just one comment about the location of the test

File: src/integrations/__spec__/testAccess.ts (hunk 1 above)
Comment (tongfa) [reply src/integrations/__spec__/testAccess.ts]: thanks, it's moved!

**Review by treardon17 — APPROVED** [review]
Thanks @tongfa!
